projects/plugins/jetpack/class.jetpack.php

```diff
@@ -5371,7 +5394,8 @@ public static function validate_onboarding_token_action( $token, $action ) {
 	 */
 	public static function permit_ssl( $force_recheck = false ) {
 		// Do some fancy tests to see if ssl is being supported.
-		if ( $force_recheck || false === ( $ssl = get_transient( 'jetpack_https_test' ) ) ) {
+		$ssl = get_transient( 'jetpack_https_test' );
+		if ( $force_recheck || false === ( $ssl ) ) {
```

Comment:
Do we care that the old code only called `get_transient()` when `$force_recheck` was set, while the new code calls it unconditionally? e.g. could that result in too many extra database hits or something?

Reply:
c787e9ca96